functions/helm-template/usage.md

```diff
@@ -0,0 +1,48 @@
+<!-- Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. -->
+
+# Helm-template Usage
+
+Kpt packages are just configuration so any solution, like the `helm template` command, which emits configuration can also be used to generate kpt packages. `Helm-template` is a kpt function which generates a new kpt package from a local Helm chart or upserts Helm chart configuration to an existing kpt package. In the context of a pipeline, these packages can then be further customized using other kpt functions.
+
+## FAQs
```

Comment:
These minor details should go last. First give an overview with a hello world example that conveys how this is used quickly

Reply:
Done